src/App.js

```diff
@@ -1,12 +1,15 @@
-import React from "react";
+import React, {useState} from "react";
```

Comment:
Great catch here! Good use of destructuring on `useState` so you can directly access that function without having to call the React class first.